frontend/app/ui/error/route.js

```diff
@@ -0,0 +1,11 @@
+import Route from "@ember/routing/route";
+import config from "customer-center/config/environment";
+
+export default class ErrorRoute extends Route {
+  setupController(controller, error) {
+    // Don't show internals in production.
+    controller.showStack = config.environment !== "production";
```

Comment:
Make this a getter in the controller - makes it easier to read

Reply:
Ok

Reply:
Done